### PR TITLE
🔨 Improve Cloudflare Docs

### PR DESCRIPTION
**Pull request by WhitigolProd**
## What Changed?

This PR is designed to directly reflect, step-by-step the process for creating a tunnel, and creating hostnames, with some minor improvements for selections and inputs.

### CODE REVIEW

File: docs/installations/reverse-proxies/cf-tunnels.mdx [hunk 1]
```diff
@@ -15,50 +15,55 @@ This guide will show you how to use Cloudflare Tunnels for SnailyCAD. With Cloud
...
+    - **Subdomain:** This is the subdomain you want to use for your SnailyCAD API. For example, if you want to use `cad.example.com` then you would enter `cad` here.
+    - **Domain:** The domain connected to your Cloudflare account. For example, if you want to use `cad.example.com` then you would enter `example.com` here.
+    - **Path:** Leave empty.
+
+    - **Service Type:** Select `HTTP`
+    - **URL:** `localhost:3000`
+
+6. Click `Save hostname`
 
 ## 4. Configuring SnailyCAD
```
Comment (Dev-CasperTheGhost) [inline docs/installations/reverse-proxies/cf-tunnels.mdx:60]:
```suggestion
## 5. Configuring SnailyCAD
```

File: docs/installations/reverse-proxies/cf-tunnels.mdx [hunk 2]
```diff
@@ -15,50 +15,55 @@ This guide will show you how to use Cloudflare Tunnels for SnailyCAD. With Cloud
...
 
-   - **Subdomain:** This is the subdomain you want to use for your SnailyCAD API. For example, if you want to use `cad.example.com` then you would enter `cad` here.
-   - **Domain:** The domain connected to your Cloudflare account. For example, if you want to use `cad.example.com` then you would enter `example.com` here.
-   - **Path:** Leave empty.
+2. Click `Save Tunnel`
+3. Head back to the Tunnels dashboard.
 
-   - **Service Type:** Select `HTTPS`
-   - **URL:** `http://localhost:3000`
+## 4. Creating a tunnel for the SnailyCAD Client
```
Comment (Dev-CasperTheGhost) [inline docs/installations/reverse-proxies/cf-tunnels.mdx:43]: For the API, the title is `Create Hostname for SnailyCAD API`, shouldn't that be the same for the client, simply replacing API with client :)?

**Review by Dev-CasperTheGhost — CHANGES_REQUESTED** [review]

File: docs/installations/reverse-proxies/cf-tunnels.mdx (hunk 2 above)
Comment (WhitigolProd) [reply docs/installations/reverse-proxies/cf-tunnels.mdx]: Ah yes, that it should. Forgot to update that! Should be fixed :)

**Review by Dev-CasperTheGhost — APPROVED** [review]

**Comment by Dev-CasperTheGhost** [comment]
Thank you!!

**Pull request merged**